---
title: Papers
author: David Gerard
date: '2020-10-06'
slug: papers
categories: []
tags: []
---



<p>Here is a list of my current papers!</p>
<div id="peer-reviewed-and-published" class="section level3">
<h3>Peer Reviewed and Published</h3>
<ul>
<li><p><strong>Gerard, D.</strong> (2020). <em>Data-based RNA-seq simulations by binomial thinning.</em> BMC Bioinformatics 21(1), 206, p. 1–14. <br> &lt;<a href="https://doi.org/10.1186/s12859-020-3450-9">doi:10.1186/s12859-020-3450-9</a>&gt; &lt;<a href="https://doi.org/10.1101/758524">bioRxiv:758524</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, &amp; Stephens, M. (2020). <em>Empirical Bayes Shrinkage and False Discovery Rate Estimation, Allowing For Unwanted Variation.</em> Biostatistics, 21(1), p. 15–32. <br>
&lt;<a href="https://doi.org/10.1093/biostatistics/kxy029">doi:10.1093/biostatistics/kxy029</a>&gt; &lt;<a href="https://arxiv.org/abs/1709.10066">arXiv:1709.10066</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong> &amp; Ferrão, L. F. V. (2020). <em>Priors for Genotyping Polyploids.</em> Bioinformatics 36(6), p. 1795–1800. <br>
&lt;<a href="https://doi.org/10.1093/bioinformatics/btz852">doi:10.1093/bioinformatics/btz852</a>&gt; &lt;<a href="https://doi.org/10.1101/751784">bioRxiv:751784</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, &amp; Stephens, M. (accepted). <em>Unifying and Generalizing Methods for Removing Unwanted Variation Based on Negative Controls.</em> Statistica sinica (in press), p. 1–42. <br>
&lt;<a href="https://doi.org/10.5705/ss.202018.0345">doi:10.5705/ss.202018.0345</a>&gt; &lt;<a href="https://arxiv.org/abs/1705.08393">arXiv:1705.08393</a>&gt;</p></li>
<li><p>Pollack, J. D., <strong>Gerard, D.</strong>, Makhatadze, G., &amp; Pearl, D. K. (2019). <em>Evolutionary conservation and structural localizations suggest a physical trace of metabolism’s progressive geochronological emergence.</em> Journal of Biomolecular Structure and Dynamics, ja, p. 1–20. <br>
&lt;<a href="https://doi.org/10.1080/07391102.2019.1679666">doi:10.1080/07391102.2019.1679666</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, Ferrão, L. F. V., Garcia, A. A. F., &amp; Stephens, M. (2018). <em>Genotyping Polyploids from Messy Sequencing Data.</em> Genetics, 210(3), p. 789–807. <br>
&lt;<a href="https://doi.org/10.1534/genetics.118.301468">doi:10.1534/genetics.118.301468</a>&gt; &lt;<a href="https://doi.org/10.1101/281550">bioRxiv:281550</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, &amp; Hoff, P. (2017). <em>Adaptive higher-order spectral estimators.</em> Electronic Journal of Statistics, 11(2), p. 3703–3737. <br>
&lt;<a href="http://doi.org/10.1214/17-EJS1330">doi:10.1214/17-EJS1330</a>&gt; &lt;<a href="https://arxiv.org/abs/1505.02114">arXiv:1505.02114</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, &amp; Hoff, P. (2016). <em>A higher-order LQ decomposition for separable covariance models.</em> Linear Algebra and its Applications, 505, p. 57–84. <br>
&lt;<a href="http://doi.org/10.1016/j.laa.2016.04.033">doi:10.1016/j.laa.2016.04.033</a>&gt; &lt;<a href="https://arxiv.org/abs/1410.1094">arXiv:1410.1094</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, &amp; Hoff, P. (2015). <em>Equivariant minimax dominators of the MLE in the array normal model.</em> Journal of Multivariate Analysis, 137, p. 32–49. <br>
&lt;<a href="http://doi.org/10.1016/j.jmva.2015.01.020">doi:10.1016/j.jmva.2015.01.020</a>&gt; &lt;<a href="https://arxiv.org/abs/1408.0424">arXiv:1408.0424</a>&gt;</p></li>
<li><p>Pollack, J. D., <strong>Gerard, D.</strong>, &amp; Pearl, D. K. (2013). <em>Uniquely Localized Intra-Molecular Amino Acid Concentrations at the Glycolytic Enzyme Catalytic/Active Centers of Archaea, Bacteria and Eukaryota are Associated with Their Proposed Temporal Appearances on Earth.</em> Origins of Life and Evolution of Biospheres, 43(2), p. 161–187. <br>
&lt;<a href="http://doi.org/10.1007/s11084-013-9331-8">doi:10.1007/s11084-013-9331-8</a>&gt;</p></li>
<li><p><strong>Gerard, D.</strong>, Gibbs, H. L., &amp; Kubatko, L. (2011). <em>Estimating hybridization in the presence of coalescence using phylogenetic intraspecific sampling.</em> BMC evolutionary biology, 11(1), 291, p. 1–12. <br>
&lt;<a href="http://doi.org/10.1186/1471-2148-11-291">doi:10.1186/1471-2148-11-291</a>&gt;</p></li>
</ul>
</div>
<div id="preprints" class="section level3">
<h3>Preprints</h3>
<ul>
<li><strong>Gerard, D.</strong> (2020). <em>Pairwise Linkage Disequilibrium Estimation for Polyploids.</em> bioRxiv, p. 1–19. <br>
&lt;<a href="https://doi.org/10.1101/2020.08.03.234476">doi:10.1101/2020.08.03.234476</a>&gt;</li>
</ul>
</div>
